### PR TITLE
Added Wallet to store and backpack, it fetch the users money. 

### PR DESCRIPTION
**Pull request by LeenaCruz**
Added Wallet to store and backpack, it fetch the users money. 
We need to give the user initial money when **signup.**

### CODE REVIEW

File: controllers/api/walletRoutes.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const {Wallet} = require('../../models');
+
+router.get('/', async (req,res) => {
+try {
+    const userId = req.session.user_id;
+console.log(userId);
+    const wallet = await Wallet.findByPk(1);
```
Comment (iKeyToLife) [inline controllers/api/walletRoutes.js:8]: Change to findOne(where: {user_id: user.session.id})

File: controllers/api/walletRoutes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const {Wallet} = require('../../models');
+
+router.get('/', async (req,res) => {
```
Comment (iKeyToLife) [inline controllers/api/walletRoutes.js:4]: Add middleware withAuth

File: controllers/api/walletRoutes.js (hunk 1 above)
Comment (LeenaCruz) [reply controllers/api/walletRoutes.js:8]: Got it

File: controllers/api/walletRoutes.js (hunk 2 above)
Comment (LeenaCruz) [reply controllers/api/walletRoutes.js:4]: Done

**Pull request closed without merging**